scripts/more_demos/demo_feeAddr_is_reserveAddr.sh

```diff
@@ -46,10 +45,8 @@ tx_from_m create-bond \
   --function-type=swapper_function \
   --function-parameters="" \
   --reserve-tokens=res,rez \
-  --reserve-address="$RESERVE" \
   --tx-fee-percentage=0.5 \
   --exit-fee-percentage=0 \
-  --fee-address="$RESERVE" \
```

Comment:
Do not forget to remove this demo. This case will no longer happen because the user will never know the reserve address when creating the bond.

Reply:
File deleted